webapp/views.py

```diff
@@ -1,9 +1,10 @@
 # Standard library
+import datetime
+import html
```

Comment:
This doesn't seem to be used anywhere?

Reply:
Removing!

Reply:
Sorry, it's not used by your code, but it seems we are using it here https://github.com/canonical/ubuntu.com/pull/13636/files#diff-eb0c47c22a26e963ac57d9840cff927d3d40d2f0889900c17e6ed717459a2db6L911

This file is becoming too big...